packages/asset-uploader/src/utils/mime.ts

```diff
@@ -0,0 +1,234 @@
+// Utils related to MIME types and the `accept` attribute value format
```

Comment:
This is a good place to add unit test for all these functions to make sure we can extend the test suite over time